writer.go

```diff
@@ -214,6 +214,9 @@ func (w *Writer) WriteAttribute(row int, field int, value interface{}) error {
 	if w.dbf == nil {
 		return errors.New("Initialize DBF by using SetFields first")
```

Comment:
Would you mind changing the existing error messages to lower case whenever you encounter them?